### PR TITLE
Add https? to logger so terminals can recognize URL

### PR DESCRIPTION
**Pull request by dirksierd**
With recognized URL you can easily open the server in default browser.
Not entirely sure if I got the SSL-part right though.

### CODE REVIEW

File: lib/goliath/runner.rb [hunk 1]
```diff
@@ -299,7 +299,8 @@ def setup_stdout_logger(log, log_format)
      def run_server
        log = setup_logger
 
-       log.info("Starting server on #{@address}:#{@port} in #{Goliath.env} mode. Watch out for stones.")
+       ssl = [nil, 'off'].include?(@server_options[:ssl]) ? false : true
```
Comment (igrigorik) [inline lib/goliath/runner.rb]: I think our OptionsParser might be misleading you a bit.. server_options[:ssl] is actually treated as a binary setting. You can see the runtime check here: https://github.com/postrank-labs/goliath/blob/master/lib/goliath/server.rb#L88

As such, I think you remove the include? line and just do the binary check in the log.info.

**Comment by igrigorik** [comment]
Looks good! One last thing, can you please squash these two commits?

**Comment by dirksierd** [comment]
There you go! If all's well that would be my first, albeit modest, OSS-contribution.

**Comment by igrigorik** [comment]
Awesome work, thank you, and congrats! 😃  👍

**Pull request merged**